src/Components/Attendee.js

```diff
@@ -1,3 +1,35 @@
-export default function Attendee() {
-  return;
-}
+import { useState } from "react";
+export default function Attendee({ attendee, event, updateEventAttendance }) {
+ 
+  const [showAttendees, setShowAttendees] = useState(false);
+
+  
+  function toggleEventAttendees() {
+    setShowAttendees(!showAttendees);
+  }
+  return (
+    <div key={attendee.id} className="attendee">
+    <p>
+      <img src={attendee.avatar} alt={attendee.firstName} />
+      {"   "}
+      <span>
+        {" "}
+        {attendee.firstName} {attendee.lastName}{" "}
+      </span>
+    </p>
+    <p>
+      <button
+        className="clickable"
+        onClick={() => updateEventAttendance(event.id, attendee.id)}
```

Comment:
You created the correct function on line 7 but passed in an incorrect function on line 23. When I change it to the correct one, all the tests pass!

Reply:
What a surprise! lol Thank You Raz.